seg_lapa/config/trainer/trainer.yaml

```
@@ -0,0 +1,11 @@
+# @package _group_
+name: trainer
+gpus: "0"
+overfit_batches: 0.0
+distributed_backend: "ddp"
+num_nodes: 1
+precision: 32
```

```suggestion
precision: 16
```
We should use mixed training by default.